pkg/endpoint/endpoint.go

```diff
@@ -41,9 +40,6 @@ import (
 	"github.com/cilium/cilium/pkg/lock"
 	"github.com/cilium/cilium/pkg/logging/logfields"
 	"github.com/cilium/cilium/pkg/mac"
-	"github.com/cilium/cilium/pkg/maps/cidrmap"
-	"github.com/cilium/cilium/pkg/maps/ctmap"
-	"github.com/cilium/cilium/pkg/maps/lxcmap"
 	"github.com/cilium/cilium/pkg/maps/policymap"
```

Comment:
Is there something different about policymap? I guess we're building up maps of the actual datapath value representation from that package.

Reply:
The endpoint structure contains a reference to `policymap.PolicyMap`.